src/app/app.module.ts

```diff
@@ -103,6 +105,8 @@ const enableDevTools = environment && environment.enableDevTools;
     { provide: HAMMER_GESTURE_CONFIG, useClass: IonicGestureConfig },
     OutcomeBehaviourMapProvider,
     LogHelper,
+    NavigationHelper,
+    NavigationStateProvider,
```

Comment:
Could these be part of the Tests Module and loaded for  those effects only instead of globally loading these